### DISCUSSION TITLE
Discord Server

### DISCUSSION THREAD

**Discussion by NoahGWood** [Spooky-Manufacturing/8Q#31] [General]
I've set up a discord server for Spooky Manufacturing, I think this will be a far better method of discussion and easier for everyone to keep up-to-date on what's happening. The invite link is https://discord.gg/kp2em98yEN

**Comment by InfamousPlatypus** [comment]
The invite has expired.

**Reply by NoahGWood** [reply]
Thanks Infamous, I've updated the link
